File: Testes de performance por disciplina/Fundamentos de Desenvolvimento com Java/3º Teste de Performance/questoes_10_12/Sphere.java
package com.bam.sandcastlejava.edu.infnet.tp3.questoes_10_12;

public class Sphere {
    private double radius;

    public Sphere(int radius) {
        this.radius = radius;
    }

    public double calculateVolume() {
        return (4.0 / 3.0) * Math.PI * (radius * radius * radius);
    }
}

File: Testes de performance por disciplina/Fundamentos de Desenvolvimento com C# (P2)/3º Teste de Performance/questoes_10-12/Sphere.cs
namespace TesteDePerformance3.questoes_10_12;

public class Sphere{
    private double _radius;

    public Sphere(double radius){
        _radius = radius;
    }
    
    public double CalculateVolume(){
        return (4.0 / 3.0) * Math.PI * (_radius * _radius * _radius);
    }
}